torch/csrc/jit/script/concrete_module_type.cpp

```diff
@@ -1,8 +1,51 @@
 #include <torch/csrc/jit/script/concrete_module_type.h>
+#include <torch/csrc/jit/pybind_utils.h>
 
 namespace torch {
 namespace jit {
 namespace script {
+
+bool isSimpleConstant(const IValue& v) {
+  return v.isBool() || v.isDouble() || v.isInt() || \
+    v.isString() || v.isNone() || v.isDevice();
+}
```

Comment:
Can we put this somewhere more central? Like in ivalue.h or even as a method on IValue

Reply:
this is used only in class constant right now, not sure if this is useful for others in general.

Reply:
I'm just worried about discoverability: someone might come across the need for the same concept in the future but won't know to look in `concrete_module_type.cpp`. If it's a method on IValue then they could see it in the class declaration or in their autocomplete

This is a nit, though, so up to you

Reply:
I think we can move this to ivalue.h when a new use case comes up

Reply:
we'll remove this part and keep this logic in python